<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VueChargementJeu</class>
 <widget class="QWidget" name="VueChargementJeu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="boutonNiveau1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Chancery L</family>
     <pointsize>48</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelChoixNiveau">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>701</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Chancery L</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Choisir un niveau</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonNiveau2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Chancery L</family>
     <pointsize>48</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonNiveau4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Chancery L</family>
     <pointsize>48</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonNiveau5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>120</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Chancery L</family>
     <pointsize>48</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonNiveau3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Chancery L</family>
     <pointsize>48</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="Line" name="separateur1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separateur2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separateur3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separateur4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>130</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separateur5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>661</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelChoixNiveau2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>661</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Chancery L</family>
     <pointsize>16</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonRetour">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>540</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>boutonNiveau1</tabstop>
  <tabstop>boutonNiveau2</tabstop>
  <tabstop>boutonNiveau3</tabstop>
  <tabstop>boutonNiveau4</tabstop>
  <tabstop>boutonNiveau5</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
